src/megaclient.cpp

```diff
@@ -3718,6 +3718,7 @@ void MegaClient::resumeResumableSyncs()
         if (!nodebyhandle(config.getRemoteNode()))
         {
             // remote node gone
+            syncConfigs->remove(config.getLocalPath());
```

Comment:
The sync should not be removed in this case. It would change the current behavior in MEGAsync (just try to remove the node relative to a synced folder and you will see the SyncConfig stays there, but disabled). Removing is not an option here. Please, disable the sync instead of remove.

Reply:
Ah yes. Will change to that. Thanks!